setup.bash

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# Setup Manubot
+# Based on https://github.com/manubot/rootstock/blob/main/SETUP.md
+# This is designed to be run from the terminal on linux
+
+# Stop on first error.
+set -e
+
+usage() {
+echo "Usage: $0 [--owner <owner-string>] [--repo <repo-string>]"
+echo "Replace OWNER and REPO details for your manuscript repo location:"
+echo "i.e. https://github.com/OWNER/REPO."
+1>&2; exit 1; }
+
+# Option strings
+SHORT=o:r:
+LONG=owner:,repo:
+
+# read the options
+OPTS=$(getopt --options $SHORT --long $LONG --name "$0" -- "$@")
+
+if [ $? != 0 ] ; then echo "Failed to parse options...exiting." >&2 ; exit 1 ; fi
+
+eval set -- "$OPTS"
+
+# extract options and their arguments into variables.
+while true ; do
+  case "$1" in
+    -o | --owner )
+      shift;
+      OWNER=$1
+      shift
+      ;;
+    -r | --repo )
+      REPO="$2"
+      shift 2
+      ;;
+    -- )
+      shift
+      break
+      ;;
+    *)
+      echo "Internal error!"
+      exit 1
+      ;;
+  esac
+done
+
+if [ -z "${OWNER}" ] || [ -z "${REPO}" ]; then
+    usage
+fi
+
+# Check Remote Repo Exists.
+echo
+while true
+do
+ read -r -p "Have you manually created https://github.com/${OWNER}/${REPO}? [y/n] " input
+
+ case $input in
+   [yY][eE][sS]|[yY])
```

Comment:
Fair point, I have made the process more interactive so it can be run without any arguments as you suggested, and also with the option for no interaction using a --yes flag.